fuse/plugins/truth_information/peak_truth.py

```diff
@@ -86,6 +120,13 @@ class PeakTruth(strax.OverlapWindowPlugin):
         help="Drift velocity of electrons in the liquid xenon [cm/ns]",
     )
 
+    photon_finding_window = straxen.URLConfig(
+        default=200,
+        type=int,
+        help="Time window [ns] that defines whether a photon is in a peak. "
+        "Peaks' start and end times are extended by this window to find photons in them.",
+    )
+
```

Comment:
I'm wondering if this window is maybe a bit too wide. We can have large peaks that are split in two and as a result, the end time of the first peak is the start time of the second peak. If we now have this window, we would put some photons into both peaks.

Reply:
You are right. It's not the same story. The left extension of peak building should be enough. Now I removed it.